test/performance/range/container_seq_write_benchmark.cpp

```diff
@@ -95,13 +95,64 @@ BENCHMARK_TEMPLATE(sequential_write, seqan3::bitcompressed_vector, seqan3::aa27)
 BENCHMARK_TEMPLATE(sequential_write, seqan3::bitcompressed_vector, seqan3::alphabet_variant<char, seqan3::dna4>);
 
 BENCHMARK_TEMPLATE(sequential_write, small_vec, char);
+BENCHMARK_TEMPLATE(sequential_write, small_vec, uint32_t);
 BENCHMARK_TEMPLATE(sequential_write, small_vec, seqan3::gap);
 BENCHMARK_TEMPLATE(sequential_write, small_vec, seqan3::dna4);
 BENCHMARK_TEMPLATE(sequential_write, small_vec, seqan3::gapped<seqan3::dna4>);
 BENCHMARK_TEMPLATE(sequential_write, small_vec, seqan3::dna15);
 BENCHMARK_TEMPLATE(sequential_write, small_vec, seqan3::aa27);
 BENCHMARK_TEMPLATE(sequential_write, small_vec, seqan3::alphabet_variant<char, seqan3::dna4>);
 
+// ============================================================================
+//  SeqAn2: sequential_write
+// ============================================================================
+
+#if SEQAN3_HAS_SEQAN2
+
+#include <seqan/sequence.h>
+
+template <template <typename...> typename container_t, typename spec_t, typename alphabet_t>
+void sequential_write2(benchmark::State & state)
+{
+    auto cont_rando = seqan3::test::generate_sequence_seqan2<alphabet_t>(10'000, 0, 0);
+    container_t<alphabet_t, spec_t> source = cont_rando;
+
+    alphabet_t a{};
+    for (auto _ : state)
+    {
+        for (auto && c : source)
+            c = a;
```

Comment:
DO_NOT_OPTIMIZE?

Reply:
It probably not necessary, but if it has side affects it will be the same as for the other benchmarks? Just a thought.